specs/lib/mocks.ts

```diff
@@ -92,6 +93,12 @@ const backendSrv = {
 
     if (!response) {
       console.error('Backend request mock not found.');
+    } else if (response instanceof Error) {
+      const defer = $q.defer();
+
+      defer.reject(response);
+
+      return defer.promise;
```

Comment:
I think these 3 lines could be condensed to `return $q.reject(response)` without the need for creating a new promise yourself

Reply:
True.

Reply:
See: https://github.com/VividCortex/grafana-datasource/pull/27